src/main/java/liquibase/ext/databricks/sqlgenerator/CreateViewGeneratorDatabricks.java

```diff
@@ -0,0 +1,64 @@
+package liquibase.ext.databricks.sqlgenerator;
+
+import liquibase.Scope;
+import liquibase.database.Database;
+import liquibase.ext.databricks.change.createView.CreateViewStatementDatabricks;
+import liquibase.ext.databricks.database.DatabricksDatabase;
+import liquibase.parser.LiquibaseSqlParser;
+import liquibase.parser.SqlParserFactory;
+import liquibase.sql.Sql;
+import liquibase.sql.UnparsedSql;
+import liquibase.sqlgenerator.SqlGeneratorChain;
+import liquibase.sqlgenerator.core.CreateViewGenerator;
+import liquibase.statement.core.CreateViewStatement;
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CreateViewGeneratorDatabricks extends CreateViewGenerator {
+
+    @Override
+    public int getPriority() {
+        return PRIORITY_DATABASE;
+    }
+
+    @Override
+    public boolean supports(CreateViewStatement statement, Database database) {
+        return super.supports(statement, database) && (database instanceof DatabricksDatabase);
```

Comment:
We might check instanceof first, because boolean statements with double signs like '&&' are "fail fast" and if DB is inapropriate there won't be a call to super.
I'm not sure if my purpose makes sence in readability perspective though.